addons/website/views/snippets/s_tabs.xml

```diff
@@ -20,13 +20,13 @@
                 </div>
                 <div class="s_tabs_content tab-content">
                     <div class="tab-pane fade show active" id="nav_tabs_content_1" role="tabpanel" aria-labelledby="nav_tabs_link_1">
-                        <div class="oe_structure oe_empty"/>
+                        <t t-call="website.s_text_block"/>
```

Comment:
I don't think you were supposed to remove the feature to be able to drop snippets in each tab, just pre-filling the tab with text. So it could indeed be a call to s_text_block, but still inside the oe_structure div I think. To confirm with SBU.

Also, when you t-call snippet templates, you have to use t-snippet-call instead of t-call otherwise there won't be the data-snippet attribute which is added automatically :+1: